packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript

```diff
@@ -0,0 +1,8 @@
+name="minecraft-launcher-deb"
+pkgname="minecraft-launcher"
+version="928"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin"
```

Comment:
```suggestion
breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app"
```

Reply:
> Also do you know if minecraft supplies a url where the version is static?

No idea

Reply:
Because that's kind of necessary for this script to work